sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -2722,6 +2722,17 @@ object SQLConf {
       .booleanConf
       .createWithDefault(false)
 
+  val TRUNCATE_TRASH_ENABLED =
+    buildConf("spark.sql.truncate.trash.enabled")
```

Comment:
quick question, do we want to have each configuration for each operation? Looks like https://github.com/apache/spark/pull/29319 targets similar stuff. Maybe it'd make more sense to have a global configuration.

Reply:
I will rework on  #29319 and make it a global configuration.

Reply:
Yep. It's too early to make it a global configuration.